### PR TITLE
Fs jul19 yevhen sag - html-form solution

### PR DESCRIPTION
**Pull request by eugeneSahach**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,7 +13,115 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit(event)">
+    <fieldset>
+      <legend>Personal information</legend>
+      <div class="field">
```
Comment (YuriiHoliuk) [inline src/index.html]: Do you really need additional div

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+fieldset {
+  margin-bottom: 20px;
+}
+
+.field {
+  margin-bottom: 10px;
+}
+
+.field:last-child {
```
Comment (YuriiHoliuk) [inline src/style.css]: can you achieve the same result with 1 selector?

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+fieldset {
```
Comment (YuriiHoliuk) [inline src/style.css]: do not style tags

**Comment by yevhen-sahach** [comment]
I've removed div tags and changed selectors, please check it out

**Pull request closed without merging**